types/express-serve-static-core/index.d.ts

```diff
@@ -488,7 +488,8 @@ export interface MediaType {
     subtype: string;
 }
 
-export type Send = (body?: any) => Response;
+// tslint:disable-next-line no-unnecessary-generics
+export type Send = <T = any>(body?: T) => Response;
```

Comment:
I can see why you'd want to make this sort of type check, but I'm not sure it ought to be associated with a particular function (i.e. `Send`).  Perhaps you'd be better off with a general purpose helper like 
```ts
function checkType<T>(t: T): T { return t; }
```